dd-sdk-android-ndk/src/test/cpp/utils/greatest/package.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Is this json necessary ?

Reply:
Not really sure as I didn't not work with C++ deps so far. I guess we could dump it as we already added the licence in our Licence.csv file